config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
I have another question:
I think It would be ok to put the file under each folder,  src/application/i18n, src/editor/i18n and so on
How can I use each file separately . Currently, I can use the "GlobalizePlugin" in the webpack.config.js file to define the path for messages for each language, I don't know yet how to load files under each folder separately. Some help would be appreciated
Thanks